documentation/source/users/arkane/credits.rst

```diff
@@ -41,11 +41,4 @@ Developers: (rmg_dev@mit.edu)
 How to Cite
 ***********
 
-A. Grinberg Dana, M.S. Johnson, J.W. Allen, S. Sharma, S. Raman, M. Liu, C.W. Gao, C.A. Grambow, M.J. Goldman,
-D.S. Ranasinghe, R.J. Gillis, A.M. Payne, Y.-P. Li, X. Dong, K.A. Spiekermann, H. Wu, E.E. Dames, Z.J. Buras,
-N.M. Vandewiele, N.W. Yee, S.S. Merchant, B. Buesser, C.A. Class, C.F. Goldsmith, R.H. West, W.H. Green,
-"Automated reaction kinetics and network exploration (Arkane):
-A statistical mechanics, thermodynamics, transition state theory, and master equation software",
-*International Journal of Chemical Kinetics* 2023, 55(6), 300-323.
-
-DOI: `10.1002/kin.21637 <https://doi.org/10.1002/kin.21637>`_
+Please refer to the `CITATIONS.bib file <https://github.com/ReactionMechanismGenerator/RMG-Py/blob/main/CITATIONS.bib>`_.
```

Comment:
Could maybe add to that file comments about when to cite what. But maybe that's not helpful, or already obvious.

Reply:
added a brief mention of which to use